lib/rules/require-newline-before-block-statements.js

```diff
@@ -97,6 +97,7 @@ module.exports.prototype = {
             errors.assert.differentLine({
                 token: prevToken,
                 nextToken: openingBrace,
+                allowExtra: true,
```

Comment:
Yeah, I didn't have this originally but it's tests were assuming that differentLine wasn't strict and only guaranteed that they were in fact on *different* lines, not that they were on touching lines.